PanoptesNetClient/PanoptesNetClient/Models/Subject.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace PanoptesNetClient.Models
+{
+    [Path("/api/subjects", "subjects")]
+    public class Subject : IResource
+    {
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        [JsonProperty("zooniverse_id")]
+        public string ZooniverseId { get; set; }
+
+        [JsonProperty("locations")]
+        public List<dynamic> Locations { get; set; }
+
+        [JsonProperty("metadata")]
+        public dynamic Metadata { get; set; }
```

Comment:
Would this not be type `Dictionary`

Reply:
I kept it dynamic for now so it would be more flexible until we know exactly what the metadata looks like.